scaaml/capture/acqktp.py

```diff
@@ -0,0 +1,226 @@
+from __future__ import division
+from __future__ import absolute_import
+
+from collections import OrderedDict
```

Comment:
This seems to be a leftover from previous iterations of my code. This import isn't used anymore (we should consider having pylint or pyflakes and yapf workflows to check PRs)

Reply:
I did a local pylint + yapf, a proper workflow is still a TODO.